pyhf/utils.py

```diff
@@ -127,27 +127,34 @@ def pvals_from_teststat(sqrtqmu_v, sqrtqmuA_v):
     return CLsb, CLb, CLs
 
 
-def runOnePoint(muTest, data, pdf, init_pars=None, par_bounds=None):
+def hypotest(poi_test, data, pdf, init_pars=None, par_bounds=None, **kwargs):
     r"""
-    Computes test statistics (and expected statistics) for a single value
-    of the parameter of interest
+    Computes test statistics (and expected statistics) for a single value of the parameter of interest
 
     Args:
-        muTest (Number or Tensor): The value of the parameter of interest (POI)
+        poi_test (Number or Tensor): The value of the parameter of interest (POI)
         data (Number or Tensor): The root of the calculated test statistic given the Asimov data, :math:`\sqrt{q_{\mu,A}}`
-        init_pars (Array or Tensor): the initial parameter values to be used for minimization
-        par_bounds (Array or Tensor): the parameter value bounds to be used for minimization
+        init_pars (Array or Tensor): The initial parameter values to be used for minimization
+        par_bounds (Array or Tensor): The parameter value bounds to be used for minimization
+
+    Keyword Args:
+        return_observed_set (bool): Bool for returning :math:`CL_{s+b}` and :math:`CL_{b}`
```

Comment:
just commenting to remind ourselved to find a better name.. the name currently suggests some notion of analogy to the expected_set which should not be implied. But can't come up with one right now

Reply:
Returning CLs set sounds clearer, since s+b and b is how you calculate CLs.

Reply:
While I agree that just about anything is better than `return_observed_set` I think that `return_cls` might be confusing as by default a CLs is already returned. I don't have any useful suggestions at the moment though, so let me come back after I finish other work today with something to contribute.

Reply:
yeah `cl_s` will be confusing. I was thinking perhaps `p_values` since this is exactly what they are

Reply:
Changed in https://github.com/diana-hep/pyhf/pull/341/commits/51992a82c5422e9c8806db905d26491371ac41b5